packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -656,11 +659,12 @@ class _ExpansionTileState extends State<ExpansionTile> with SingleTickerProvider
   Widget _buildChildren(BuildContext context, Widget? child) {
     final ThemeData theme = Theme.of(context);
     final ExpansionTileThemeData expansionTileTheme = ExpansionTileTheme.of(context);
+    final Color backgroundColor = _backgroundColor.value ?? expansionTileTheme.backgroundColor ?? Colors.transparent;
     final ShapeBorder expansionTileBorder = _border.value ?? const Border(
-            top: BorderSide(color: Colors.transparent),
-            bottom: BorderSide(color: Colors.transparent),
-          );
-    final Clip clipBehavior = widget.clipBehavior ?? expansionTileTheme.clipBehavior ?? Clip.none;
+      top: BorderSide(color: Colors.transparent),
+      bottom: BorderSide(color: Colors.transparent),
+    );
+    final Clip clipBehavior = widget.clipBehavior ?? expansionTileTheme.clipBehavior ?? Clip.antiAlias;
```

Comment:
Maybe make the last clause be
```dart
(isShapeProvided ? clip.antiAlias : Clip.none)
```

Since `clip.antiAlias` comes with a cost. And then document the fact the default for clipBehavior depends on whether of not a shape was provided.

Reply:
`clipBehavior` is never used without shape. 

It returns a `DecoratedBox` when no shape is provided for two reasons:
1. We don't need to add` Material` widget when not necessary`.
2. Previously `Container` used the `clipBehavior` but it was effective as the ink splash is actually rendered by the underling `ListTile` widget. 

<img width="689" alt="Screenshot 2024-01-19 at 20 00 39" src="https://github.com/flutter/flutter/assets/48603081/939daa2e-7559-4489-be19-71728ebc00a0">

Reply:
I see, looks good then!